CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+- GraphSage OGBN example to mirror the DGL implementation from leaderboard ([#9386](https://github.com/pyg-team/pytorch_geometric/pull/9386))
```

Comment:
What does "mirror the DGL implementation" mean? If we copied any part of the DGL implementation, we should comment in the example script.